### DISCUSSION TITLE
How to contribute to the StudioServer code?

### DISCUSSION THREAD

**Discussion by raja-jamwal** [prisma/prisma#25486] [Q&A]
### Question

Hello,

I’m trying to build a use case for Prisma Studio, and I’m currently reviewing the contribution guide here & found following for prisma studio cli:
https://github.com/prisma/prisma/blob/8e14920742f1aa11bd201c073b31ec7cd9d03d3a/packages/cli/src/Studio.ts#L16.

However, I’m having trouble locating the code for @prisma/studio-server. I’ve already checked under packages/ and packages/cli.

Any guidance or a getting-started guide for Prisma Studio would be greatly appreciated.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @raja-jamwal 👋 

Thank you for your interest in contributing to Prisma Studio. Please note that Prisma Studio, including the @prisma/studio-server, is not open-source, which is why you might have difficulty locating its code in the public repositories.

If you have suggestions or feature requests for Prisma Studio, I encourage you to share them by creating an issue on the [Prisma Studio GitHub repository](https://github.com/prisma/studio). This is the best way to ensure that your ideas are considered by the Prisma team.

Feel free to reach out if you have any other questions!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
